NeoML/src/Dnn/BaseLayer.cpp

```diff
@@ -417,7 +417,8 @@ void CBaseLayer::reshape()
 
 class CRunOnceTimer {
 public:
-	CRunOnceTimer( bool enable, IMathEngine& mathEngine, int& hitCount, IPerformanceCounters::CCounter::TCounterType& result );
+	CRunOnceTimer( bool enable, IMathEngine& mathEngine, int& hitCount, 
+		IPerformanceCounters::CCounter::TCounterType& result, bool isOnlyTime = false);
```

Comment:
This is an internal class (it isn't located in public header). And it's used _only for time measurement_.

So, there's no need for parameter in ctor. Simply call `mathEngine.CreatePerformanceCounters( true )`.